cmake/packaging.cmake

```diff
@@ -197,20 +212,63 @@ function(generateInstallTargets)
     # .
     install(PROGRAMS "$<TARGET_FILE:osqueryd>" DESTINATION . RENAME osqueryi.exe)
 
-    file(COPY "${CMAKE_SOURCE_DIR}/tools/deployment/osquery.example.conf" DESTINATION "${CMAKE_BINARY_DIR}/package/wix")
-    install(FILES "${CMAKE_BINARY_DIR}/package/wix/osquery.example.conf" DESTINATION . RENAME osquery.conf)
+    if("${PACKAGING_SYSTEM}" STREQUAL "NuGet")
+      set(win_packager "nuget")
+      # Grab the Chocolatey management scripts
+      file(COPY "${CMAKE_SOURCE_DIR}/tools/deployment/chocolatey/tools" DESTINATION "${CMAKE_BINARY_DIR}/package/${win_packager}")
+      install(DIRECTORY "${CMAKE_BINARY_DIR}/package/${win_packager}/tools" DESTINATION .)
+
+      # Chocolatey requires a LICENSE.txt and VERIFICATION.txt file in all 
+      # packages containing .exe, .msi, or .zip files.
+      file(COPY "${CMAKE_SOURCE_DIR}/LICENSE" DESTINATION "${CMAKE_BINARY_DIR}/package/${win_packager}")
+      install(FILES "${CMAKE_BINARY_DIR}/package/${win_packager}/LICENSE" DESTINATION . RENAME LICENSE.txt)
 
-    file(COPY "${CMAKE_SOURCE_DIR}/tools/wel/osquery.man" DESTINATION "${CMAKE_BINARY_DIR}/package/wix")
-    install(FILES "${CMAKE_BINARY_DIR}/package/wix/osquery.man" DESTINATION .)
+      file(WRITE "${CMAKE_BINARY_DIR}/package/${win_packager}/VERIFICATION.txt" "
+VERIFICATION
```

Comment:
Can you put this file into ./tools (into an appropriate sub folder) and then copy it similar to how CMake is copying other files.

Reply:
I could, the issue I was originally having, and why this is here, is I wanted to grab the hashes of the built binaries and have them be included in this file. For now I'll just remove the pre-computed hashes and maybe that can be a follow-up for us, as I don't think it's required.